cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -572,3 +588,111 @@ impl<
 		fee
 	}
 }
+
+/// A `HandleFee` implementation that takes fees from `ExportMessage` XCM instructions
+/// to Snowbridge and holds it in a receiver account. Burns the fees in case of a failure.
+pub struct XcmExportFeeToSnowbridge<
```

Comment:
Since we're going to use this type in about 5 different BridgeHub runtimes, lets move it to a new crate named `snowbridge-runtime-common`, located in our own Snowbridge repo, in this directory: `parachain/primitives/runtime-common`.

Then we can the runtimes depend on our runtime-common crate.

Reply:
Addressed in https://github.com/Snowfork/polkadot-sdk/pull/23/commits/e3bd0e291d2727d05b09dd2d792531f5e95a0dd3